template/package.json

```diff
@@ -19,37 +19,38 @@
   "author": "",
   "devDependencies": {
     {{#lint}}
-    "eslint": "^5.10.0",
+    "eslint": "^6.1.0",
     {{/lint}}
     {{#jest}}
-    "jest": "^23.6.0",
-    "jest-cli": "^23.6.0",
+    "jest": "^24.8.0",
+    "jest-cli": "^24.8.0",
     {{/jest}}
-    "moleculer-repl": "^0.5.3"
+    "moleculer-repl": "^0.5.7"
   },
   "dependencies": {
     {{#apiGW}}
-    "moleculer-web": "^0.8.0",
+    "moleculer-web": "^0.8.5",
     {{/apiGW}}
     {{#if_eq transporter "NATS"}}
-    "nats": "^1.0.1",
+    "nats": "^1.3.0",
     {{/if_eq}}
     {{#if_eq transporter "MQTT"}}
-    "mqtt": "^2.18.8",
+    "mqtt": "^3.0.0",
     {{/if_eq}}
     {{#if_eq transporter "AMQP"}}
-    "amqplib": "^0.5.3",
+    "amqplib": "^0.5.5",
     {{/if_eq}}    
     {{#if_eq transporter "STAN"}}
-    "node-nats-streaming": "0.0.51",
+    "node-nats-streaming": "^0.2.6",
     {{/if_eq}}    
     {{#if_eq transporter "Kafka"}}
-    "kafka-node": "3.0.1",
+    "kafka-node": "^4.1.3",
     {{/if_eq}}    
     {{#redis}}
-    "ioredis": "^4.2.0",
+    "ioredis": "^4.14.0",
+    "redlock": "^4.0.0",
```

Comment:
Redlock is not required (e.g. Redis can be the transporter)

Reply:
> Redlock is not required (e.g. Redis can be the transporter)

yeah, fixed